prow/plugins/project/project.go

```diff
@@ -0,0 +1,348 @@
+	successClearingProjectMsg = "You have successfully removed this issue/PR from project %s."
+	clearKeyword              = "clear"
+	projectNameToIDMap        = make(map[string]int)
+)
+
+type githubClient interface {
+	CreateComment(owner, repo string, number int, comment string) error
+	ListTeamMembers(id int, role string) ([]github.TeamMember, error)
+	GetRepos(org string, isUser bool) ([]github.Repo, error)
+	GetRepoProjects(owner, repo string) ([]github.Project, error)
+	GetOrgProjects(org string) ([]github.Project, error)
+	GetProjectColumns(projectID int) ([]github.ProjectColumn, error)
+	CreateProjectCard(columnID int, projectCard github.ProjectCard) (*github.ProjectCard, error)
+	GetColumnProjectCard(columnID int, cardNumber int) (*github.ProjectCard, error)
+	MoveProjectCard(projectCardID int, newColumnID int) error
+	DeleteProjectCard(projectCardID int) error
+	TeamHasMember(teamID int, memberLogin string) (bool, error)
+}
+
+func init() {
+	plugins.RegisterGenericCommentHandler(pluginName, handleGenericComment, helpProvider)
+}
+
+func getMaintainerTeam(pluginConfigs plugins.ProjectConfig, org string, repo string) (int, string) {
+	for orgName, orgConfig := range pluginConfigs.Orgs {
+		if org == orgName {
+			// look for repo level configs first because repo level config overrides org level configs
+			for repoName, repoConfig := range orgConfig.Repos {
+				if repo == repoName {
+					return repoConfig.MaintainerTeamID, repoConfig.MaintainerTeamName
+				}
+			}
+			return orgConfig.MaintainerTeamID, orgConfig.MaintainerTeamName
+		}
+	}
+	return 0, ""
+}
+
+func getColumnMap(pluginConfigs plugins.ProjectConfig, org string, repo string) map[string]string {
+	for orgName, orgConfig := range pluginConfigs.Orgs {
+		if org == orgName {
+			for repoName, repoConfig := range orgConfig.Repos {
+				if repo == repoName {
+					return repoConfig.ProjectColumnMap
+				}
+			}
+			return orgConfig.ProjectColumnMap
+		}
+	}
+	return nil
+}
+
+func getOrgColumnMap(pluginConfigs plugins.ProjectConfig, org string, repo string) map[string]string {
+	for orgName, orgConfig := range pluginConfigs.Orgs {
+		if org == orgName {
+			return orgConfig.ProjectColumnMap
+		}
+	}
+	return nil
+}
+
+func helpProvider(config *plugins.Configuration, enabledRepos []string) (*pluginhelp.PluginHelp, error) {
+	configInfo := map[string]string{}
+	for _, repo := range enabledRepos {
+		parts := strings.Split(repo, "/")
+		if len(parts) != 2 {
+			return nil, fmt.Errorf("invalid repo in enabledRepos: %q", repo)
+		}
+		if maintainerTeamID, maintainerTeamName := getMaintainerTeam(config.Project, parts[0], parts[1]); maintainerTeamID != 0 && maintainerTeamName != "" {
+			configInfo[repo] = fmt.Sprintf(projectTeamMsg, maintainerTeamName, maintainerTeamID)
+		} else {
+			configInfo[repo] = "There are no maintainer team specified for this repo or its org."
```

Comment:
This means when someone has no idea about this plugin and does not configure it, they get a negative message. This could be confusing -- maybe let's just omit here.